common/types/layer.go

```diff
@@ -41,52 +43,64 @@ func GetLayersPerEpoch() uint32 {
 
 // GetEffectiveGenesis returns when actual proposals would be created.
 func GetEffectiveGenesis() LayerID {
-	return NewLayerID(atomic.LoadUint32(&effectiveGenesis))
+	return LayerID(atomic.LoadUint32(&effectiveGenesis))
 }
 
 // NewLayerID creates LayerID from uint32.
 func NewLayerID(value uint32) LayerID {
```

Comment:
is this still needed?

Reply:
Technically no, I can remove it and replace it with `types.LayerID()` instead.

Reply:
Removed